src/components/Tree/child.js

```diff
@@ -67,9 +100,20 @@ Child.propTypes = {
     isLoading: PropTypes.bool,
     icon: PropTypes.node,
     children: PropTypes.array,
-    onExpandCollapse: PropTypes.func,
-    onSelect: PropTypes.func,
+    /** The action triggered when the user clicks in the tree node checkbox. */
+    onNodeCheck: PropTypes.func,
+    /** The action triggered when the user clicks in the tree node expand or collapse button. */
```

Comment:
these descriptions are only needed in index.js since they are used to create the documentation but only are taken from there